src/handlers/request-handlers/root-request-handler.ts

```diff
@@ -26,14 +26,13 @@ export const rootRequestHandler = (request: Request, response: Response, next: N
       software: packageJson.repository.url,
       version: packageJson.version,
       limitation: {
-            max_message_length: settings.network.maxPayloadSize,
+            max_payload_size: settings.network.maxPayloadSize,
```

Comment:
does not conform to the protocol: https://github.com/nostr-protocol/nips/pull/163/files#diff-e2738305ed5e49cb3b4cb1295e8028c000b203b4e74a4c5fe599dc884e7a92d3R44

Reply:
Oh, I see.  I'll remove from this PR, but try to update the docs to match the reference in another PR.